src/components/layout/__tests__/Toolbar-test.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import Toolbar from '../Toolbar';
+import { shallow } from 'enzyme';
+import FontAwesome from 'react-fontawesome';
+import { MusitField } from '../../formfields';
+import { Button } from 'react-bootstrap';
+
+describe('Toolbar', () => {
+  it('should create a toolbar with left toolbar item active', () => {
+    const showLeft = true;
+    const leftLabel = 'Left';
+    const clickShowLeft = () => false;
+    const rightLabel = 'Right';
+    const showRight = false;
+    const clickShowRight = () => false;
+    const wrapper = shallow(<Toolbar
+      clickShowLeft={clickShowLeft}
+      clickShowRight={clickShowRight}
+      labelLeft={leftLabel}
+      labelRight={rightLabel}
+      showLeft={showLeft}
+      showRight={showRight}
+      onSearchChanged={() => false}
+      placeHolderSearch="Search here"
+      searchValue=""
+    />);
+    expect(wrapper.contains(
+      <Button active={showLeft} onClick={clickShowLeft}>
+        <FontAwesome name={'check-square-o'} />
+        {' '}
+        {leftLabel}
+      </Button>
+    )).toBe(true);
+    expect(wrapper.contains(
+      <Button active={showRight} onClick={clickShowRight}>
+        <FontAwesome name={'square-o'} />
+        {' '}
+        {rightLabel}
+      </Button>
+    )).toBe(true);
+    expect(wrapper.contains('Right')).toBe(true);
```

Comment:
That test is actually unnecessary, so i can remove it. Its asserted in the above tests